launch/launch.go

```diff
@@ -27,6 +27,7 @@ type launch struct {
 	runner      runner
 }
 
+// EnvVar is a map for environment variables
 type EnvVar map[string]string
```

Comment:
[imo] Is it better to define an alias type for `map[string]interface{}` like `EnvVar`?